libs.ts

```diff
@@ -876,6 +876,23 @@ export const libs: Lib[] = [
 		timeouts: 'Yes',
 		modals: 'Yes'
 	},
+  	{
+		name: 'pincer',
+		url: 'https://github.com/Pincer-org/pincer',
+		language: 'Python',
+		apiVer: 9,
+		gwVer: 9,
+		slashCommands: 'Yes',
+		buttons: 'Yes',
+		selectMenus: 'Yes',
+		threads: 'Yes',
+		guildStickers: 'Yes',
+		contextMenus: 'No',
```

Comment:
it seems to have some context menu support:
https://github.com/Pincer-org/Pincer/blob/7b00f52908ef7768785cd9c23d711892c0b9a71e/pincer/objects/app/command_types.py#L7-L27

is it not fully implemented?

Reply:
Those are just the Enum values, I don't think it is implemented. I asked it in their Discord server and they didn't seem to respond with a affirmative 'yes'.